repos/system_upgrade/el7toel8/actors/restrictedpcisscanner/actor.py

```diff
@@ -0,0 +1,31 @@
+from leapp.actors import Actor
+from leapp.libraries.actor import restrictedpcisscanner
+from leapp.models import RestrictedPCIDevices
+from leapp.tags import ExperimentalTag, FactsPhaseTag, IPUWorkflowTag
+
+
+class RestrictedPCIsScanner(Actor):
+    """
+    Provides data about restricted (unsupported or/and unavailable) devices.
+
+    The data sources priority set up is as following:
+    1. The actor tries to get the data from /etc/leapp/files
+    2. Then from the online service
+    3. As a fallback mechanism the data comes from the locally stored
```

Comment:
Same here - no fallback anymore, please update comment

Reply:
resolved in https://github.com/oamg/leapp-repository/pull/602/commits/06e630c346b953bbe1dc9d26628be33ae194abfa